src/isomorphic/deprecated/__tests__/cloneWithProps-test.js

```diff
@@ -30,6 +30,25 @@ describe('cloneWithProps', function() {
     onlyChild = require('onlyChild');
     cloneWithProps = require('cloneWithProps');
     emptyObject = require('emptyObject');
+    spyOn(console, 'error');
+  });
+
+  it('should warn because it is deprecated', function() {
+    var Parent = React.createClass({
+      render: function() {
+        return (
+          <div>
+            {cloneWithProps(onlyChild(this.props.children), {})}
+          </div>
+        );
+      },
+    });
+    ReactTestUtils.renderIntoDocument(<Parent><div /></Parent>);
+    expect(console.error.argsForCall.length).toBe(1);
+    expect(console.error.calls[0].args[0]).toContain(
```

Comment:
Let's still to using `argsForCall` here instead of using that and `calls[].args[]`. I think `argsForCall[0][0]` will be what you want.